lib/block-supports/dimensions.php

```diff
@@ -88,9 +110,9 @@ function gutenberg_skip_spacing_serialization( $block_type ) {
 
 // Register the block support.
 WP_Block_Supports::get_instance()->register(
-	'spacing',
+	'dimensions',
```

Comment:
This is the sort of thing I brought up at https://github.com/WordPress/gutenberg/pull/32499/files#r686747072 

Isn't this problematic with existing `spacing` supports? What happens when core (WordPress 5.8) registers support for `spacing` and the plugin does not overwrite it as before but instead registers a different thing called `dimensions`?

Reply:
It might make more sense to keep things anchored to `spacing` for now.

Reply:
Thanks for the explanation @nosolosw, I think I understand my error now in not appreciating that core also register support under the same key.

I'll create a quick PR to change the key above from `dimensions` back to `spacing` and add an inline comment explaining why it is still `spacing`. 

Regarding the [comments on the PR introducing height block support](https://github.com/WordPress/gutenberg/pull/32499/files/6c89606eb108b92da166b999deb6c63c59c2b464#r686747072) I think there may be some confusion there, at least there is from my side, so I'll reply in more detail on that PR.

Reply:
Fix is up in: https://github.com/WordPress/gutenberg/pull/34030

Thank you for catching this one!

Reply:
Thanks for the quick turnaround, as always!